### DISCUSSION TITLE
Feature request? Unifi Express

### DISCUSSION THREAD

**Discussion by MiranoVerhoef** [fabianishere/udm-iptv#292] [Ideas]
Hey @fabianishere. First i want to thank you for supporting this. It has been a pleasure to have my UDM Pro running with this script.

I recently bought a Unifi Express (Unifi UX) for my brother. Is this supported yet on the newest release?)

Also, would it be possible to make a "Supported Devices" tab within the repository README so we can see which device is supported and since which version?

Thanks for responding! :)

**Comment by dmutsaers** [comment]
I think it should be possible, I assume the UXG-Lite and the UX are based on the same hardware.

**Comment by dmutsaers** [comment]
Keep in mind that the required firmware needed to support IGMP Proxy hasn't been released for the UX (UniFi Express) yet.

**Comment by dmutsaers** [comment]
UniFi OS - Express 3.2.5 | Added support for IGMP Proxy
https://community.ui.com/releases/UniFi-OS-Express-3-2-5/6822b208-31a7-4c72-9e94-3180bc660e26

**Comment by jessemeijers** [comment]
I got it to work on the Unifi Express. I did use the Early Access 3.2.5 version of the OS but I am thinking this is not necessary. I only installed that to try the built-in IGMP proxy, but that didn't seem to work (I think it doesn't support multiple VLANs on the WAN).

The problem I had was that the settop box would keep giving Error 563 (which means it cannot get an IP address). This in turn means that the VLAN on the WAN isn't working correctly.

My setup:
- FTTH XGSPON KPN
- FTTH connects to WAN port on Unifi Express (UE)
- LAN port on UE connects to Unifi Switch
- Settop box KPN connects to Unifi Switch
- Separate VLAN (ID=4) for IPTV on LAN (linked this to the port the settop box is connected to on the switch)

IPTV guide for Unifi Express:
- Run the guide as per the instructions (running 'udm-iptv diagnose' will give an error that eth8 and iptv devices cannot be found)
- Log in using SSH
- Use VI to edit /etc/udm-iptv.conf
- Replace 'eth8' with 'eth1'
- Exit VI
- Open Unifi mobile app (note I use this, since the Provision option is not always available on the web client somehow)
- Provision the UE
- Go back to your SSH session and try 'udm-iptv diagnose' again. Should now work.

I hope this helps some people, as I was banging my head against this for a while until I ran the diagnose and figured that the UE obviously doesn't have eth8, only eth0 (LAN) and eth1 (WAN).

**Comment by dmutsaers** [comment]
I think the flex mini doesn't support IGMP.
________________________________
Van: ricardo0h ***@***.***>
Verzonden: vrijdag 14 juni 2024 23:06
Aan: fabianishere/udm-iptv ***@***.***>
CC: dmutsaers ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [fabianishere/udm-iptv] Feature request? Unifi Express (Discussion #292)


Unfortunately I cannot get it to work. I have mostly the same setup only I have the setup box connected to a flex mini which in turn is connected to the UniFi 16 switch. Could that be of influence? I have set the iptv network on the port on the flex mini which connects to the setup box.
